README.md

```diff
@@ -315,9 +315,10 @@ Type: String
 
 Default: `/var/log/aws-routed-eni/plugin.log`
 
-Valid Values: `stdout` or a file path
+Valid Values: `stderr` or a file path
```

Comment:
`stderr` will work. However, the logs are written to file descriptor which user cannot check/retrieve. IMO, it is redundant to write plugin logs to both `stdout` and or `stderr`

Reply:
Yeah thats right but just in case if there is a use case, I left it as stderr.

Reply:
Make sense but I still think we should call it out explicitly or else users will start complaining that plugin logs are missing.